tiup/tiup-mirror-reference.md

```diff
@@ -0,0 +1,311 @@
+            "sig": "{signature-by-index-key-1}",                # The signed part of this file by this private key.
+        },
+        ...
+        {
+            "keyid": "{id-of-root-key-N}",                      # The ID of the Nth private key that participates in the signature.
+            "sig": "{signature-by-root-key-N}"                  # The signed part of this file by this private key.
+        }
+    ],
+    "signed": {
+        "_type": "timestamp",                                   # The file type.
+        "expires": "{expiration-date-of-this-file}",            # The expiration time of the file. After the file expires, the client will reject the file.
+        "meta": {                                               # The information of snapshot.json.
+            "/snapshot.json": {
+                "hashes": {
+                    "sha256": "{sum-of-sha256}"                 # snapshot.json's sha256.
+                },
+                "length": {length-of-json-file}                 # The length of snapshot.json.
+            }
+        },
+        "spec_version": "0.1.0",                                # The specified version followed by this file. If the file structure is changed in the future, the version number needs to be upgraded. The current version number is 0.1.0.
+        "version": {N}                                          # The version number of this file. You need to overwrite timestamp.json every time you update the file, and set its version to N + 1.
+```
+
+## Client workflow
+
+The client uses the following logic to ensure that the files downloaded from the mirror are safe:
+
++ A `root.json` file is included with the binary when the client is installed.
++ The running client performs the following tasks based on the existing `root.json`:
+    1. Obtain the version from `root.json` and mark it as `N`.
+    2. Request `{N+1}.root.json` from the mirror. If the request is successful, use the public key recorded in `root.json` to verify whether the file is valid.
+    3. Request `timestamp.json` from the mirror and use the public key recorded in `root.json` to verify whether the file is valid.
+    4. Check whether the checksum of `snapshot.json` recorded in `timestamp.json` matches the checksum of the local `snapshot.json`. If the two do not match, request the latest `snapshot.json` from the mirror and use the public key recorded in `root.json` to verify whether the file is valid.
+    5. Obtain the version number `N` of the `index.json` file from `snapshot.json` and request `{N}.index.json`. Then use the public key recorded in `root.json` to verify whether the file is valid.
+    6. For components such as `tidb.json` and `tikv.json`, the client obtains the version numbers `N` of the components from `snapshot.json` and requests `{N}.{component}.json`. Then the client uses the public key recorded in `index.json` to verify whether the file is valid.
```

Comment:
```suggestion
    5. Obtain the version number `N` of the `index.json` file from `snapshot.json` and request `{N}.index.json` from the mirror. Then use the public key recorded in `root.json` to verify whether the file is valid.
    6. For components such as `tidb.json` and `tikv.json`, the client obtains the version numbers `N` of the components from `snapshot.json` and requests `{N}.{component}.json` from the mirror. Then the client uses the public key recorded in `index.json` to verify whether the file is valid.
```
@lucklove Please help confirm if `{N}.index.json` and `{N}.{component}.json` files are requested from the mirror. Thanks!

Reply:
LGTM